### DISCUSSION TITLE
Type Casting

### DISCUSSION THREAD

**Discussion by enricofengler** [modular/modular#433] [Q&A] [answered]
Hey @all,

I have a newbie question... How are types casted in general? A simple cast of f32 to si32 seems not work with the common python methods.

**Comment by gryznar** [comment, marked as answer]
Here is partial explaination, why "Python methods" currently does not work: https://github.com/modularml/mojo/issues/428 (comments)

If I am not wrong, it is going to change in the future

**Reply by enricofengler** [reply]
Thanks for the quick response.
